src/itinerary/Itinerary.svelte

```diff
@@ -3,12 +3,15 @@
   import Map from "./Map.svelte";
   import { Collection } from "sveltefire";
 
-  export const addExcursion = () => {
-    $voyage.collection("excursions").doc().set({ name: "test" });
+  let excursionName = "Excursion Name";
+  const addExcursion = () => {
+    $voyage.collection("excursions").doc().set({ name: excursionName });
   };
 </script>
 
+<input bind:value={excursionName} />
```

Comment:
You should have excursionName start out as an empty string and add a placeholder. 
RIght now the user has to delete the text "Excursion Name" before inputting the name